### DISCUSSION TITLE
[Cask Request] Spark Mail App

### DISCUSSION THREAD

**Discussion by jvanderen1** [Homebrew/discussions#3774] [Casks] [answered]
## 💡 Request

There is now an updated version of Spark written using Electron (different from MAS version).

URL: https://sparkmailapp.com/

Similar request, but was for MAS version: https://github.com/Homebrew/homebrew-cask/issues/29175

**Comment by p-linnane** [comment, marked as answer]
This is already available in `homebrew-cask`. It was added last week as `readdle-spark`. See https://github.com/Homebrew/homebrew-cask/pull/132831.

**Reply by jvanderen1** [reply]
Ohh thanks!
